integration_tests/run.sh

```diff
@@ -1,14 +1,36 @@
 #!/bin/bash
-# script for jenkins to run tests on polar2grid
+# Script for jenkins to run tests on polar2grid.
 
+export PATH="/usr/local/texlive/2019/bin/x86_64-linux":$PATH
 cd "$WORKSPACE"
-mkdir -p integration_tests/jenkins_p2g_env
+# Activate conda for bash.
+/data/users/davidh/miniconda3/bin/conda init bash
+# Restart the shell to enable conda.
+source ~/.bashrc
+conda remove -y -n jenkins_p2g_env --all
```

Comment:
Why do you remove the environment first? I suppose it doesn't work consistently otherwise?

My hope was that the `conda env update` below would do whatever was necessary to update the existing environment. Based on what I know about conda the way you are doing it now is probably actually faster (because of how the package dependencies get resolved).

Reply:
I won't need to remove it every time, there were conda-pack errors and I wasn't sure if the environment was corrupted at some point. Removing the environment fixes this temporarily, but I will find a long term fix.